proxy-node-chart/charts/verify-service-provider/values.yaml

```diff
@@ -0,0 +1,58 @@
+# Default values for verify-service-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: govukverify/verify-service-provider
+  tag: "latest"
```

Comment:
Not a requirement, but we probably should be pinning that in some way or another.